spyder/app/mainwindow.py

```diff
@@ -827,7 +827,11 @@ def create_edit_action(text, tr_text, icon):
                                         _("Fullscreen mode"),
                                         triggered=self.toggle_fullscreen,
                                         context=Qt.ApplicationShortcut)
-        self.register_shortcut(self.fullscreen_action, "_",
+        if sys.platform == 'darwin':
+            self.fullscreen_action.setEnabled(False)
+            self.fullscreen_action.setToolTip("Disabled for macOS")

```

Comment:
Let's improve this tooltip a bit. My suggestion is:

    _("Disabled for macOS because it provides its own fullscreen functionality")

Reply:
How about _("For fullscreen mode use macOS built-in feature").

Reply:
Ok, sounds good to me. Please implement your suggestion instead.